docs/listings/example_workflows/basic_workflow.yml

```diff
@@ -0,0 +1,15 @@
+name: My workflow
+on:
+  push:
+  pull_request:
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Starknet Foundry
+        uses: foundry-rs/setup-snfoundry@v3
+      - name: Check versions
+        run: |
+          snforge --version
+          sncast --version
```

Comment:
It's possible to check the installed version by extending `setup-snfoundry` output, this is redundant

Reply:
But we want to ensure that the binary is added to path and can be run. I've limited it to only run `snforge --version`